src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -97,6 +97,25 @@ object ScalafixPlugin extends AutoPlugin {
     def scalafixConfigSettings(config: Configuration): Seq[Def.Setting[_]] =
       inConfig(config)(
         Seq(
+          scalacOptions := {
```

Comment:
What happens in builds that do `scalacOptions :=` overriding the settings from plugins? My experience is that quite many builds do this with `scalacOptions`.